pkgs/top-level/all-packages.nix

```diff
@@ -15413,6 +15419,8 @@ in
 
     qca-qt5 = callPackage ../development/libraries/qca-qt5 { };
 
+    qcsxcad = callPackage ../applications/science/electronics/qcsxcad { };
```

Comment:
Isn't this an application?

Reply:
Not it is a library, I also had this suprise.
Should I move it to dev/libs ?

Reply:
It doesn't matter if upstream considers it a library or an application, it matters if we have another qt package that uses it as an input. And we do: `appcsxcad` and `openems`.